fusestore/store/src/dfs/distributed_fs.rs

```diff
@@ -52,7 +52,7 @@ impl FileSystem for Dfs {
 
         // update meta, other store nodes will be informed about this change and then pull the data to complete replication.
 
-        let req = ClientRequest {
+        let req = LogEntry {
```

Comment:
What is the rationale behind this renaming? Is this for raft?

Reply:
Yes it is an entry in the raft log.
There are already several term that are similar to `ClientRequest` literally. 🤔

Reply:
I see, is it similar to below in etcd:
https://github.com/etcd-io/etcd/blob/2a0f8f073892ea5ea9a4a467dbbc16d1576fd084/raft/raftpb/raft.pb.go#L266 ?

Reply:
Yes, `LogEntry == etcd.Entry.Data`

Reply:
Got it. No wonder LogEntry does not have  Index or Term field.